libini2.c

```diff
@@ -482,7 +482,9 @@ static struct ini_loop * ini_loop_new(char *buf_with_loop, char *loop_name)
 	loop->first = first;
 	loop->inc = inc;
 	loop->last = last;
-	snprintf(loop->var, sizeof(loop->var), "<%s>", var);
+	if (snprintf(loop->var, sizeof(loop->var), "<%s>", var) == sizeof(loop->var))
```

Comment:
I'm not sure if this check is correct ; technically, `snprintf()` returns the number of characters written (minus the null char) if the buffer was sufficient ; so, any positive value return is fine, and any negative is an error;

Reply:
From the snprintf() man page
> If the output was truncated due to this limit, then the return value is the number of characters (excluding the terminating null byte) which would have been written to the final string if enough space had been available.  Thus, a return value of size or more means that the output was truncated.

Based on that the check should be `>=`.

But if this is just about silencing the warning might as well just change the size of var from 128 to 126.